### ISSUE TITLE
STORY MAP: Hyperlink in chapter field doesn't meet the A11y color contrast requirements

### ISSUE THREAD

**Issue by Hanal-code** [techmatters/terraso-web-client#758]
## Description
The global hyperlink color needs to be adjusted for the dark background in Story map. 
The hyperlink chapter description field doesn't meet the A11y color contrast requirements, 4.5:1 (Level AA).
Change the color to `#71B8D2` and add `text-decoration: underline` to make it distinguishable in a paragraph. 

## Steps To Reproduce

1. Go to URL:https://app.staging.terraso.net/tools/story-maps/new
2. Add a chapter
3. Enter a hyperlink to chapter 1 
4. Notice the blue link color(global color) is not meeting the A11y color contrast ratio requirement 
5. 
## Expected behavior

<img width="275" alt="image" src="https://user-images.githubusercontent.com/92342777/224354681-58c9148f-1601-4486-9d1f-17e563ab5dfc.png">

**Comment by josebui** [comment]
@Hanal-code This should be fixed on staging now

**Comment by Hanal-code** [comment]
The fix has been verified.
